test/ability_intimidate.c

```diff
@@ -23,7 +23,7 @@ SINGLE_BATTLE_TEST("Intimidate (opponent) lowers player's attack after switch ou
         {
             ABILITY_POPUP(opponent, ABILITY_INTIMIDATE);
             ANIMATION(ANIM_TYPE_GENERAL, B_ANIM_STATS_CHANGE, player);
-            MESSAGE("Foe Arbok's Intimidate cuts Wobbuffet's attack!");
+            MESSAGE("Foe Arbok's ability cut \nWobbuffet's Attack!");
```

Comment:
This is just ugly. The current test system allows for in-game like breaks to be interpreted as a space in the test name, which is much cleaner. This looks like a hacky solution.
I saw that your string is set as "cut " (with a space at the end), which is unneeded. You should be able to add a space to the string if needed.

Reply:
Ohhh, it's the spaces causing the issues, not the newlines... OK, will adjust!